app/screens/reusable/Acknowledgement/AcknowledgementTitle/AcknowledgementTitle.props.ts

```diff
@@ -0,0 +1,21 @@
+import { StackScreenProps, StackNavigationProp } from '@react-navigation/stack';
+import { ImageSourcePropType } from 'react-native';
+
+import { MergedStackParams } from '../../../../navigation/stacks/mergedParams';
+
+type NavigationProps = StackNavigationProp<
+  MergedStackParams,
+  'AcknowledgementTitle'
+>;
+type RouteProps = StackScreenProps<MergedStackParams, 'AcknowledgementTitle'>;
+
+export interface IAcknowledgementTitleScreenProps
+  extends NavigationProps,
+    RouteProps {}
+
+export interface IAcknowledgementTitleRoute {
+  title: string;
+  image: ImageSourcePropType;
+  backgroundImage: ImageSourcePropType;
+  onSubmit: () => void;
+}
```

Comment:
this interface should be nested in `.types.ts`
because in `props.ts1 we should store only and only props
you can change the filenames across the project from `props.ts` to `types.ts` and put there all types if you want